### PR TITLE
Added scroll to top button with progress

### PR DESCRIPTION
**Pull request by TheNikhilRaj**
@4darsh-Dev 
Added scroll to top button with progress!
#80 
Capture

### CODE REVIEW

**Comment by TheNikhilRaj** [comment]
please help me in the deployment?
give me the old code of settings.py and urls.py code which i can put it for deployment.
@4darsh-Dev

File: django-web-app/cogniguard/settings.py [hunk 1]
```diff
@@ -151,33 +111,17 @@
     messages.ERROR: 'danger',
 }
 
-# email backend configuration
```
Comment (4darsh-Dev) [inline django-web-app/cogniguard/settings.py:154]: why have you removed email configuration, 
See the latest file of main branch
fix it

File: django-web-app/cogniguard/urls.py [hunk 2]
```diff
@@ -1,40 +1,23 @@
...
 admin.site.site_header = "CogniGuard Admin"
 admin.site.site_title = "CogniGuard Admin Portal"
-
-
 urlpatterns = [
     path('admin/', admin.site.urls),
     # path("logout/", RedirectView.as_view(url="/admin/logout/")),
     path("", include("home.urls")),
-
-    # for rendering reset password cnf template
```
Comment (4darsh-Dev) [inline django-web-app/cogniguard/urls.py:23]: same issue, 
you've removed some paths required for password reset which have been added in recent commit. 
fix it.

**Review by 4darsh-Dev — CHANGES_REQUESTED** [review]
## Some minor changes required
1. Your changes in settings.py and urls.py doesn't match with latest code from main branch, 
2. Issue might be due to not syncing the repo
3. Ensure that code written in settings. and urls.py remain same as main branch latest code in github repo

**Comment by 4darsh-Dev** [comment]
@TheNikhilRaj check for code in cogniguard github repo, latest from main branch

**Comment by TheNikhilRaj** [comment]
@4darsh-Dev I have done this same thing but giving error ! What should do??

**Comment by 4darsh-Dev** [comment]
⚠️[URGENT]⚠️ Reminder for GSSoC 24' Contributors Please join our CogniGuard Project #general GSSoC'24 Discord channel:
 Here you can discuss your queries and doubts with your mentors. 
 See #109 issues section.

**Comment by 4darsh-Dev** [comment]
⚠️[URGENT]⚠️ Reminder for GSSoC 24' Contributors Please join our CogniGuard Project #general GSSoC'24 Discord channel:
 Here you can discuss your queries and doubts with your mentors. 
 See #109 issues section.

**Comment by TheNikhilRaj** [comment]
@4darsh-Dev  it's done!

**Comment by 4darsh-Dev** [comment]
@TheNikhilRaj i;ve reviewd already , fix the changes 
can you see the diff now
##ss
Screenshot 2024-06-26 222241


Screenshot 2024-06-26 222241

**Comment by 4darsh-Dev** [comment]
Screenshot 2024-06-26 222756

**Review by 4darsh-Dev — CHANGES_REQUESTED** [review]
fix now

**Comment by TheNikhilRaj** [comment]
> Screenshot 2024-06-26 222756

check the video clip as mentioned above 
@4darsh-Dev

**Review by 4darsh-Dev — COMMENTED** [review]
## See the vercel preview of your fork link and the main repository, 
and tell me where is the error ?
@TheNikhilRaj 

https://github.com/4darsh-Dev/CogniGaurd/assets/109789509/140f051c-0654-4aaa-bfc0-fd022e055326

**Comment by TheNikhilRaj** [comment]
@4darsh-Dev  Give me the old code of settings.py and urls.py on that basis i can do changes in the error!

**Comment by 4darsh-Dev** [comment]
@TheNikhilRaj 
navigate to the project dir : django-web-app/cogniguard/settings.py 
similar for urls.py 
-- and add look for changes

**Comment by TheNikhilRaj** [comment]
> @TheNikhilRaj 
> navigate to the project dir : django-web-app/cogniguard/settings.py 
> similar for urls.py 
> -- and add look for changes

Project dir? @4darsh-Dev

**Comment by 4darsh-Dev** [comment]
@TheNikhilRaj  yes, 
https://github.com/4darsh-Dev/CogniGaurd/blob/main/django-web-app/cogniguard/settings.py

**Pull request closed without merging**